docs/kms_secretlock.md

```diff
@@ -0,0 +1,143 @@
+# Aries New Secure KMS, SecretLock and Crypto capabilities
+
+The aries-framework-go project has a mandate to protect keys from being compromised during usage or storage of keys. 
```

Comment:
We should just talk about the goal rather than Aries-framework-go mandates.

Reply:
updated text